src/components/hand-controls.js

```diff
@@ -218,6 +218,16 @@ module.exports.Component = registerComponent('hand-controls', {
         el.setAttribute('pico-controls', controlConfiguration);
         el.setAttribute('windows-motion-controls', controlConfiguration);
         el.setAttribute('hp-mixed-reality-controls', controlConfiguration);
+
+        // Pico4, at least on Wolvic, needs a different rotation
+        // offset for the hand model. Pico Browser claims to use
+        // oculus controllers instead; will load oculus-touch-controls
+        // and does not require this adjustment.
+        el.addEventListener('controllerconnected', function (evt) {
```

Comment:
There's already a `onControllerConnected` method in the component we should put that logic there.

Reply:
I can, however I need to make sure the adjustment is applied only the first time the event fires, or hands will rotate everytime I put the headset down and come back again later, hence the "once true" on the handler.

Could use a flag to be set once instead... would you prefer something like that?

Reply:
best is probably remove it the rotation on `onControllerDisconnected`

Reply:
I have tried a different approach: I have now moved the logics to compute the various offsets to the hand model out of the model load handler and into the controller-connected handler.

https://github.com/aframevr/aframe/pull/5534/commits/b0adabf2fa3ccd4fec769bc711b203879a64b5f7

I have updated the demo links